integration-libs/cdc/organization/administration/cdc-administration.module.ts

```diff
@@ -31,6 +31,7 @@ import { CdcUserListService } from './cdc-user-list.service';
               provide: UserListService,
               useExisting: CdcUserListService,
             },
+            { provide: B2BUserService, useClass: CdcB2BUserService },
```

Comment:
[QUESTION]
Is the provider in the line 42 still needed?

Note: Analogical question applies to line 59